src/main/java/xyz/vegaone/easytrackingv2/service/UserStoryService.java

```diff
@@ -31,17 +33,21 @@ public UserStory createUserStory(UserStory userStory) {
     }
 
     public UserStory getUserStory(Long id) {
-        UserStoryEntity userStoryEntity = userStoryRepo.findOne(id);
 
-        if (userStoryEntity == null) {
+        Optional<UserStoryEntity> userStoryOptional = userStoryRepo.findById(id);
+
+        if (userStoryOptional.isPresent()) {
```

Comment:
Optional has a neat feature, called `orElseThrow`. Please use that. See the comment on the `ProjectService` class